recommence/Config.py

```diff
@@ -19,3 +21,35 @@ def get_staging_path(self) -> str:
 
     def get_staging_data_path(self) -> str:
         return f'{self.get_staging_path()}/{self.data_file}'
+
+@dataclass
+class ReporterConfig:
+    '''
+    config type can be one of the following:
+    no_report: No report
+    logger:  Through the logger
+    file: To user specified file
+    sql: To user specified sql database location
+    '''
+    type: str
```

Comment:
Let's change this to `types: List[str]`, then iterate through multiple writer backends when writing so that the user can choose multiple places for reporting (e.g. I'd imagine a common configuration would be to write to disk and write to a sysadmin database).